### DISCUSSION TITLE
Can figures be included into the slides?

### DISCUSSION THREAD

**Discussion by AustinMarstaller** [jeertmans/manim-slides#287] [Q&A] [answered]
I am a math phd candidate and would like to know if figures can be created using manim-slides. Currently, the main system to use is Beamer which allows creating figures.

**Comment by jeertmans** [comment, marked as answer]
Hello, what do you mean by created?

You can include images with Manim using an [`ImageMobject`](https://docs.manim.community/en/stable/reference/manim.mobject.types.image_mobject.ImageMobject.html), an [`SVGMobject`](https://docs.manim.community/en/stable/reference/manim.mobject.svg.svg_mobject.SVGMobject.html), or call some LaTeX code with [`Tex`](https://docs.manim.community/en/stable/reference/manim.mobject.text.tex_mobject.Tex.html) that further includes images, like with Beamer.
